### ISSUE TITLE
enhancement suggestion: link multiple joomla sites

### ISSUE THREAD

**Issue by EZS-JD** [salesagility/SuiteCRM#640]
I love the general idea of AOP with Joomla.

We regularly start new projects which need a unique web portal where a customer interaction such as what AOP allows is needed. As it currently only allows one such Joomla Site to be linked to SuiteCRM, we won't be able to make much use of it for this purpose.

It would be great if I could easily set up multiple Joomla-sites and link them with SuiteCRM so that a SuiteCRM Contact can be a Portal user in one or many of my multiple Joomla Sites!

**Comment by ghermans** [comment]
:+1:

**Comment by chris001** [comment]
Very interesting. The Wordpress - SuiteCRM Customer Portal (by biztechconsultancy) is almost identical to the official SuiteCRM Joomla - SuiteCRM AdvancedOpen Portal.
The Wordpress one supports multisites ie multiple wordpress sites.  
https://wordpress.org/plugins/sugarcrmsuitecrm-customer-portal/faq/
Should look at the code and see how they implement multiple wordpress websites and possibly do the same for multiple joomla websites - this is the beauty of open source PHP.
https://downloads.wordpress.org/plugin/sugarcrmsuitecrm-customer-portal.zip

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [enhancement suggestion: link multiple joomla sites](https://trello.com/c/xqT8bM9N/136-enhancement-suggestion-link-multiple-joomla-sites)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
